pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/fraser/barnepensjon/Adopsjon.kt

```diff
@@ -17,12 +23,12 @@ object Adopsjon {
             }
             paragraph {
                 text(
-                    Language.Bokmal to "Barnepensjonen din opphører fra <dato>.",
+                    Language.Bokmal to "Barnepensjonen din opphører fra ${virkningsdato.format()}.",
```

Comment:
Fungerer dette? `.format` burde returnere et StringExpression, så jeg ville sett for meg at det du får her når malen rendrer er toString invokasjon på Expression-objektet.

```
textExpr(Bokmal to "Barnepensjonen din opphører fra".expr() + virkningsdato.format())
```

Reply:
Oops. Godt sett!